Problems/TwoPunctures/two_punctures_robin_sphere_hp_mgpc_newton_petsc_cactus.c

```diff
@@ -495,7 +493,7 @@ problem_init
     if (p4est->mpirank == 0){
       printf("[D4EST_OUTPUT]: Norms in cubic region only\n");
     }
-    d4est_output_norms
+    d4est_norms_norms
```

Comment:
norms_norms should go to norms, but d4est_output_norms will be removed in favour of d4est_norms_save, so this can be done later I guess